frontend/packages/console-app/src/components/user-preferences/UserPreferenceCheckboxField.tsx

```diff
@@ -16,6 +18,7 @@ const UserPreferenceCheckboxField: React.FC<UserPreferenceCheckboxFieldProps> =
   trueValue,
   falseValue,
   defaultValue,
+  description,
```

Comment:
do we need `description` , is it being used?

Reply:
Currently, we are not using it. But it can be used in features as we have a description property defined in extention.